StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1627DocumentationTextMustNotBeEmpty.cs

```diff
@@ -42,7 +47,7 @@ internal class SA1627DocumentationTextMustNotBeEmpty : DiagnosticAnalyzer
         /// </summary>
         public const string DiagnosticId = "SA1627";
         private const string Title = "Documentation text must not be empty";
-        private const string MessageFormat = "TODO: Message format";
+        private const string MessageFormat = "The documentation text within the {0} tag must not be empty.";
```

Comment:
:bulb: The `{0}` here should be in single quotes since it's a replaceable element whose contents appear in the source code itself. For example: `'{0}'`.